xcm/pallet-xcm-benchmarks/src/generic/benchmarking.rs

```diff
@@ -496,6 +496,23 @@ benchmarks! {
 		assert_eq!(executor.origin(), &Some(X1(alias).relative_to(&universal_location)));
 	}
 
+	export_message {
+		let mut executor = new_executor::<T>(Here.into_location());
+		let (network, destination) = T::bridged_destination()?;
+		let expected_message = Xcm(vec![TransferAsset {
```

Comment:
@acatangiu 
what about to change:
 `let (network, destination) = T::bridged_destination()?;`
to
 `let (network, destination, xcm) = T::bridged_destination()?;`

then runtime can customize their XCM

Reply:
Technically speaking, this field shouldn't need to be filled, as what the benchmark really does is to only benchmark the execution of the `ExportMessage` instruction. I also doubt that the contents of the `xcm` field even contributes to the overall weight of the message.